jobs/async-upload/requirements.txt

```diff
@@ -1,7 +1,7 @@
 # This file is automatically generated by poetry export. Do not edit manually.
 aiohappyeyeballs==2.6.1 ; python_version >= "3.11" and python_version < "4.0"
 aiohttp-retry==2.9.1 ; python_version >= "3.11" and python_version < "4.0"
-aiohttp==3.12.15 ; python_version >= "3.11" and python_version < "4.0"
+aiohttp==3.13.2 ; python_version >= "3.11" and python_version < "4.0"
```

Comment:
with the manual bump of the requirements.txt this complete the dependency bump of aiohttp which is used by the MR py client (specifically the code generated REST client it wraps) and internally as a transitive to make requests

/lgtm
/approve